tests/tst_qPref.qml

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0
+import QtQuick 2.6
+import QtTest 1.2
+import org.subsurfacedivelog.mobile 1.0
+
+TestCase {
+	name: "qPref"
+
+	SsrfPrefs {
+		id: prefs
+	}
+
+	function test_register() {
+		var x = prefs.mobile_version
+		compare(x, prefs.mobile_version)
```

Comment:
isn't that a tautology?
I'm not quite sure what this tests, TBH

Reply:
it test that the type is registred and can be accessed (which happened to be the problem with my previous PR). Normally you will see test files with a lot more test cases, I kept this simple in order to make it easy to copy.